release-notes/6.0/6.0.0.md

```diff
@@ -0,0 +1,234 @@
+        /// </summary>
+        public string Value { get ; }
+    }
+}
+```
+
+JSON datatype is supported by the driver for reading, writing, streaming and performing bulk copy operations.
+
+Implemented in PRs: [#2916](https://github.com/dotnet/SqlClient/pull/2916), [#2892](https://github.com/dotnet/SqlClient/pull/2892), [#2891](https://github.com/dotnet/SqlClient/pull/2891), [#2880](https://github.com/dotnet/SqlClient/pull/2880), [#2882](https://github.com/dotnet/SqlClient/pull/2882), [#2829](https://github.com/dotnet/SqlClient/pull/2829), [#2830](https://github.com/dotnet/SqlClient/pull/2830)
+
+#### Introducing SqlClientDiagnostics
+
+`SqlClientDiagnostic` is now available as a strongly-typed collection of key-value pairs that can be captured by consuming applications.
+
+```csharp
+// Class that provides strongly-typed collection of key-value pairs for SqlClient diagnostic objects.
+public abstract class SqlClientDiagnostic : System.Collections.Generic.IReadOnlyList<System.Collections.Generic.KeyValuePair<string, object>>
+{
+    // A guid value used to correlate before, after and error events.
+    public System.Guid OperationId;
+    
+    // The name of the operation.
+    public string Operation;
+
+    // The timestamp of the event.
+    public long Timestamp;
+
+    // The number of elements in the collection.
+    public int Count;
+
+    // The element at the specified index in the read-only list.
+    public System.Collections.Generic.KeyValuePair<string, object> this[int index];
+    
+    // An enumerator that can be used to iterate through the collection.
+    public System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, object>> GetEnumerator();
+}
+```
+
+Implemented in PR: [#2226](https://github.com/dotnet/SqlClient/pull/2226)
+
+#### Added Support for Connection Overrides in `OpenAsync()` API
+
+The default behavior of `SqlConnection.OpenAsync()` can be overridden to disable the ten-second delay and automatic connection retries triggered by transient errors.
+
+```csharp
+using(SqlConnection sqlConnection = new SqlConnection("Data Source=(local);Integrated Security=true;Initial Catalog=AdventureWorks;"))
+{
+    await sqlConnection.OpenAsync(SqlConnectionOverrides.OpenWithoutRetry, cancellationToken);
+}
+```
+
+Implemented in PR: [#2433](https://github.com/dotnet/SqlClient/pull/2433)
+
+#### Other Additions
+
+- Added support for .NET 9 [#2946](https://github.com/dotnet/SqlClient/pull/2946)
+- Added localization in Czech, Polish, and Turkish [#2987](https://github.com/dotnet/SqlClient/pull/2987)
+- Added `TokenCredential` object to take advantage of token caching in `ActiveDirectoryAuthenticationProvider`. [#2380](https://github.com/dotnet/SqlClient/pull/2380)
+- Added `DateOnly` and `TimeOnly` support to `DataTable` as a structured parameter. [#2258](https://github.com/dotnet/SqlClient/pull/2258)
```

Comment:
And SqlDataRecord (apparently)

Reply:
@benrr01 see my PR here, and related issues. Apparently my fix in #2258 also impacted SqlDataRecord

Reply:
Ok, I think this rewriting of the addition explains it well enough.